### ISSUE TITLE
Linter silently fails on null value error

### ISSUE THREAD

**Issue by wolfkarl** [awslabs/aws-cfn-lint-atom#32]
```yml
AWSTemplateFormatVersion: "2010-09-09"
Resources:
  # Unsupported Type "AWS::EC2::Instance1"
  MyEC2Instance1:
    Type: "AWS::EC2::Instance1"
      Metadata:
```

This minimal config makes the linter fail silently -- no errors are shown.

Running `cfn-lint` manually returns `E0000 Null value at line [...]`

**Comment by rdkls** [comment]
This is because `--ignore-bad-template` is supplied to `cfn-lint`
https://github.com/aws-cloudformation/cfn-lint-atom/blob/60f95f6a74c989c19e8fbe46cd884efaf71ef1df/lib/atom-cfn-lint.js#L168

@kddejong would you be OK with us removing this? Can make PR
